Website/src/rendering/src/components/ShopCommon/IfCommerceEnabled.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
```

Comment:
What is the need for this component? Is it to check if commerce is enabled and then wrap the children in a provider?

Reply:
It renders components conditionally based on wheather or not commerce is enabled.

Reply:
Personally I find it a bit confusing, in part because of the existence of the isCommerceEnabled helper, but if @jflheureux has no objection to this component, then everything else seems good!

Reply:
I think I understand where the need for this component came. The ShopNavigation is not a child of the UserProvider/Provider/OcProvider inside the ShopLayout component.

Another solution could be to move the ShopNavigation as child of these providers in the ShopLayout:
```

  // Show shop content if commerce is enabled, otherwise show error message
  const shopContent = isCommerceEnabled ? (
    <UserProvider>
      <Provider store={reduxStore}>
        <OcProvider>
          <header>
            <ShopNavigation />
          </header>
          <main>
            <HeaderCdpMessageBar />
            <div className="shop-main-container">{props.children}</div>
          </main>
        </OcProvider>
      </Provider>
    </UserProvider>
  ) : (
    <p className="shop-integration-error">
      Shop pages are currently disabled because the commerce integration is not configured
    </p>
  );

  return (
    <>
      <Head>
        <link rel="icon" href="/favicon.ico" />
      </Head>
      {shopContent}
      <footer>
        <Footer {...footerProps} />
      </footer>
    </>
  );
```